src/Filter/CustomFieldsFilterService.php

```diff
@@ -17,12 +17,11 @@ public function __construct(CustomFieldRepoService $repo)
         $this->repo = $repo;
     }
 
-    public function applyFilter($filterform, QueryBuilder $qb, $firstRootAlias = null)
+    public function applyFilter($entityClass, $filterform, QueryBuilder $qb, $firstRootAlias = null)
```

Comment:
do not introduce argument $entityClass, instead see `$entityClass = ...` below
With this, we avoid a bc break.

When we **need** it as **optional** argument, do it as with $firstRootAlias.